code/game/machinery/autolathe.dm

```diff
@@ -19,6 +19,7 @@
 	var/busy = 0
 
 	var/mat_efficiency = 1
+	var/coeff = 1

```

Comment:
Has this entirely replaced `mat_efficiency`, or what purpose do the two now serve? In-code comments are very helpful for this sort of thing.

Additionally, if this is a 1-or-0 thing, it should be TRUE/FALSE.

Reply:
Numeric value, but I just realised we don't actually need it defined in this line anyway.

Reply:
That doesn't explain the purpose of the variable, which was the more important part of my last comment.

Reply:
The comments in autolathe.dm (line 9) and the other autolathe.dm (line 86 and 255) do.